Sources/CorrilySDK/Misc/Extensions/URLSession.swift

```diff
@@ -11,7 +11,9 @@ import Foundation
 extension URLSession {
   func data(for request: URLRequest) async throws -> (Data, URLResponse) {
      try await withCheckedThrowingContinuation { continuation in
-       let task = self.dataTask(with: request, completionHandler: { data, response, error in
+       
+         // Shouldn't we use await dataTask here and remove completion handler?
```

Comment:
This extension provides async/await syntax for URLSession.data(for request: URLRequest) to support iOS 13 and 14. We can remove it when we target iOS 15+.

Reply:
aa, ok, let's keep supporting iOS 13